src/server/pegasus_mutation_duplicator.cpp

```diff
@@ -235,7 +244,8 @@ void pegasus_mutation_duplicator::duplicate(mutation_tuple_set muts, callback cb
         }
 
         if (batch_count == muts.size() ||
-            batch_bytes >= dsn::replication::FLAGS_duplicate_log_batch_bytes) {
+            batch_bytes >= dsn::replication::FLAGS_duplicate_log_batch_bytes ||
+            batch_bytes >= dsn::replication::FLAGS_dup_max_allowed_write_size) {
```

Comment:
`batch_bytes` is still possible to be larger than `max_allowed_write_size` on backup cluster even if `dup_max_allowed_write_size` equales to `max_allowed_write_size`, isn't it?